src/sbvr-api/sbvr-utils.coffee

```diff
@@ -995,7 +1056,15 @@ exports.executeStandardModels = executeStandardModels = (tx, callback) ->
 		throw err
 	.nodeify(callback)
 
-exports.addHook = (method, apiRoot, resourceName, callbacks) ->
+exports.addHook = (method, apiRoot, resourceName, hooks) ->
+	pureHooks = _.mapValues hooks, (hook) ->
+		{
+			RUN: hook
```

Comment:
It might be better to call this `HOOK` to avoid any possible confusion with `PRERUN`/`POSTRUN`